advntr/genome_analyzer.py

```diff
@@ -7,26 +7,31 @@
 from advntr.sam_utils import extract_unmapped_reads_to_fasta_file
 from advntr.vntr_finder import VNTRFinder
 
+from sys import stdout
```

Comment:
Please don't use stdout directly. We set the target for print command in advntr_command.py and using print we can either write to a file (determined by `--outfile`) or stdout (if no output file is specified). Here when you use combination of both different part of output go to different streams (e.g. output file or stdout).